mula/scheduler/server/errors.py

```diff
@@ -0,0 +1,33 @@
+import functools
+
+import fastapi
+import pydantic
+
+from scheduler import storage
+
+
+def exception_handler(func):
```

Comment:
Instead of implementing this as a decorator, you could register the exception handlers as shown here:

https://fastapi.tiangolo.com/tutorial/handling-errors/#install-custom-exception-handlers

This would remove the need to decorate each route with the handler, and remove the need to import the decorator in those files.